### ISSUE TITLE
Table form elements and buttons are missing labels

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-filemanager#114]
Table form elements from the file content view are missing labels, as well as the dropdown button from the download action.
